src/screens/wallet/send-bottom-sheet/send-bottom-sheet.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC, useCallback, useState } from 'react';
+import { Button, Text, TouchableOpacity } from 'react-native';
+
+import { BottomSheet, BottomSheetProps } from '../../../components/bottom-sheet/bottom-sheet';
+import { StyledTextInput } from '../../../components/styled-text-input/styled-text-input';
+import { EmptyFn } from '../../../config/general';
+import { useShelter } from '../../../shelter/use-shelter.hook';
+import { WalletStyles } from '../wallet.styles';
+import { SendBottomSheetStyles } from './send-bottom-sheet.styles';
+
+interface Props extends BottomSheetProps {
+  from: string;
+  onClose: EmptyFn;
+  balance?: string;
+}
+
+export const SendBottomSheet: FC<Props> = ({ from, isOpen, onClose, onDismiss, balance }) => {
+  const [amount, setAmount] = useState('0');
+  const [recipient, setRecipient] = useState('tz1L21Z9GWpyh1FgLRKew9CmF17AxQJZFfne');
+  const { send: send$ } = useShelter();
+
+  const send = useCallback(() => {
+    send$(from, amount, recipient);
+  }, [send$, from, amount, recipient]);
```

Comment:
This `useCallback` is redundant, as the call of `useCallback` also requires some time to execute.   
Do not use `useCallback` and `useMemo` until there is an issue with performance.